gradle/dependencies.gradle

```diff
@@ -1,158 +1,160 @@
-      testEspressoIntents    : "androidx.test.espresso:espresso-intents:${version.espressoVersion}",
-
-      // unit test
-      junit                  : "junit:junit:${version.junit}",
-      mockito                : "org.mockito:mockito-core:${version.mockito}",
-      mockk                  : "io.mockk:mockk:${version.mockkVersion}",
-      hamcrest               : "org.hamcrest:hamcrest-junit:${version.hamcrest}",
-      commonsIO              : "commons-io:commons-io:${version.commonsIO}",
-      robolectric            : "org.robolectric:robolectric:${version.robolectric}",
-      json                   : "org.json:json:${version.json}",
-
-      // play services
-      gmsLocation            : "com.google.android.gms:play-services-location:${version.gmsLocation}",
-
-      firebaseCore           : "com.google.firebase:firebase-core:${version.firebaseCore}",
-      crashlytics            : "com.crashlytics.sdk.android:crashlytics:${version.crashlytics}",
-
-      multidex               : "androidx.multidex:multidex:${version.multidex}",
-
-      errorprone             : "com.google.errorprone:error_prone_core:${version.errorprone}"
-  ]
-
-  pluginVersion = [
-      checkstyle       : '8.2',
-      pmd              : '5.8.1',
-      errorprone       : '0.0.13',
-      coveralls        : '2.8.1',
-      spotbugs         : '1.3',
-      gradle           : '3.5.1',
-      dependencyGraph  : '0.3.0',
-      dependencyUpdates: '0.20.0',
-      kotlin           : '1.3.50',
-      license          : '0.8.5',
-      jacoco           : '0.8.2',
-      playPublisher    : '1.2.2',
-      googleServices   : '4.2.0',
-      crashlytics      : '1.26.1'
-  ]
-
-  pluginDependencies = [
-      gradle           : "com.android.tools.build:gradle:${pluginVersion.gradle}",
-      kotlin           : "org.jetbrains.kotlin:kotlin-gradle-plugin:${pluginVersion.kotlin}",
-      checkstyle       : "com.puppycrawl.tools:checkstyle:${pluginVersion.checkstyle}",
-      license          : "com.jaredsburrows:gradle-license-plugin:${pluginVersion.license}",
-      spotbugs         : "gradle.plugin.com.github.spotbugs:gradlePlugin:${pluginVersion.spotbugs}",
-      coveralls        : "org.kt3k.gradle.plugin:coveralls-gradle-plugin:${pluginVersion.coveralls}",
-      errorprone       : "net.ltgt.gradle:gradle-errorprone-plugin:${pluginVersion.errorprone}",
-      dependencyGraph  : "com.vanniktech:gradle-dependency-graph-generator-plugin:${pluginVersion.dependencyGraph}",
-      dependencyUpdates: "com.github.ben-manes:gradle-versions-plugin:${pluginVersion.dependencyUpdates}",
-      jacoco           : "com.github.ben-manes:gradle-versions-plugin:${pluginVersion.jacoco}",
-      playPublisher    : "com.github.triplet.gradle:play-publisher:${pluginVersion.playPublisher}",
-      googleServices   : "com.google.gms:google-services:${pluginVersion.googleServices}",
-      crashlytics      : "io.fabric.tools:gradle:${pluginVersion.crashlytics}"
-  ]
+    androidVersions = [
```

Comment:
Minor thing: Could we keep current formatting so we can easily 👀 what was changed / added?